### PR TITLE
Translate docs/reference/kubernetes-api/common-definitions/node-selector-requirement.md into Chinese 

### LINKED ISSUE #32024: [zh]Translate docs/reference/kubernetes-api/common-definitions/node-selector-requirement.md into Chinese
**This is a Feature Request**

**What would you like to be added**
   Translate docs/reference/kubernetes-api/common-definitions/node-selector-requirement.md into Chinese
   
**Why is this needed**
     Not translated yet

**Comments**
/language zh

### PR DESCRIPTION
**Pull request by shixiuguo**
Fixes #32024

### CODE REVIEW

File: content/zh/docs/reference/kubernetes-api/common-definitions/node-selector-requirement.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+The file is auto-generated from the Go source code of the component using a generic
+[generator](https://github.com/kubernetes-sigs/reference-docs/). To learn how
+to generate the reference documentation, please read
+[Contributing to the reference documentation](/docs/contribute/generate-ref-docs/).
+To update the reference content, please follow the 
+[Contributing upstream](/docs/contribute/generate-ref-docs/contribute-upstream/)
+guide. You can file document formatting bugs against the
+[reference-docs](https://github.com/kubernetes-sigs/reference-docs/) project.
+-->
+<!--
```
Comment (howieyuen) [inline content/zh/docs/reference/kubernetes-api/common-definitions/node-selector-requirement.md:36]: leave the translation in comment block...

File: content/zh/docs/reference/kubernetes-api/common-definitions/node-selector-requirement.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+该文件是使用通用Go源码自动生成的组件 [生成器](https://github.com/kubernetes-sigs/reference-docs/)。
```
Comment (howieyuen) [inline content/zh/docs/reference/kubernetes-api/common-definitions/node-selector-requirement.md:37]: 中英文混合时，英文前后需要添加空格，句末除外
```suggestion
该文件是使用通用 Go 源码自动生成的组件[生成器](https://github.com/kubernetes-sigs/reference-docs/)。
```

File: content/zh/docs/reference/kubernetes-api/common-definitions/node-selector-requirement.md [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+学习如何生成参考文档，请阅读 [贡献参考文档](/docs/contribute/generate-ref-docs/)。
```
Comment (howieyuen) [inline content/zh/docs/reference/kubernetes-api/common-definitions/node-selector-requirement.md:38]: except API reference page, other links should be localized 
```suggestion
学习如何生成参考文档，请阅读[贡献参考文档](/zh/docs/contribute/generate-ref-docs/)。
```

File: content/zh/docs/reference/kubernetes-api/common-definitions/node-selector-requirement.md [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+更新参考内容，请按照 [贡献上游](/docs/contribute/generate-ref-docs/contribute-upstream/)。
```
Comment (howieyuen) [inline content/zh/docs/reference/kubernetes-api/common-definitions/node-selector-requirement.md:39]:
```suggestion
更新参考内容，请按照[贡献上游](/zh/docs/contribute/generate-ref-docs/contribute-upstream/)指南。
```

File: content/zh/docs/reference/kubernetes-api/common-definitions/node-selector-requirement.md [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+针对文档格式化，可以参考 [参考文档](https://github.com/kubernetes-sigs/reference-docs/)。
```
Comment (howieyuen) [inline content/zh/docs/reference/kubernetes-api/common-definitions/node-selector-requirement.md:40]:
```suggestion
你可以针对[参考文档](https://github.com/kubernetes-sigs/reference-docs/)提交文档格式错误。
```

File: content/zh/docs/reference/kubernetes-api/common-definitions/node-selector-requirement.md [hunk 6]
```diff
@@ -0,0 +1,96 @@
...
+-->
+
+
+`import "k8s.io/api/core/v1"`
+
+<!--
+A node selector requirement is a selector that contains values, a key, and an operator that relates the key and values.
+-->
+   节点选择器是一个需要包含键、值以及一个将键与值关联起来的运算符的选择器。
```
Comment (howieyuen) [inline content/zh/docs/reference/kubernetes-api/common-definitions/node-selector-requirement.md:49]:
```suggestion
节点选择器要求包含键、值和关联键和值的运算符的选择器。
```

File: content/zh/docs/reference/kubernetes-api/common-definitions/node-selector-requirement.md [hunk 7]
```diff
@@ -0,0 +1,96 @@
+---
+api_metadata:
+  apiVersion: ""
+  import: "k8s.io/api/core/v1"
+  kind: "NodeSelectorRequirement"
+content_type: "api_reference"
+description: "节点选择器是一个需要包含键、值以及一个将键与值关联起来的运算符的选择器"
```
Comment (howieyuen) [inline content/zh/docs/reference/kubernetes-api/common-definitions/node-selector-requirement.md:7]:
```suggestion
description: "节点选择器要求包含键、值和关联键和值的运算符的选择器"
```

File: content/zh/docs/reference/kubernetes-api/common-definitions/node-selector-requirement.md [hunk 8]
```diff
@@ -0,0 +1,96 @@
...
+   节点选择器是一个需要包含键、值以及一个将键与值关联起来的运算符的选择器。
+
+<hr>
+
+<!--
+- **key** (string), required
+  
+  The label key that the selector applies to.
+-->
+- **key**（字符串），必选项
```
Comment (howieyuen) [inline content/zh/docs/reference/kubernetes-api/common-definitions/node-selector-requirement.md:58]:
```suggestion
- **key**（string），必选
```

File: content/zh/docs/reference/kubernetes-api/common-definitions/node-selector-requirement.md [hunk 9]
```diff
@@ -0,0 +1,96 @@
...
+
+   key是一种被选择器使用的标签。
```
Comment (howieyuen) [inline content/zh/docs/reference/kubernetes-api/common-definitions/node-selector-requirement.md:60]:
```suggestion
   选择器适用的标签键。
```

File: content/zh/docs/reference/kubernetes-api/common-definitions/node-selector-requirement.md [hunk 10]
```diff
@@ -0,0 +1,96 @@
...
+
+   Possible enum values:
+   - `"DoesNotExist"`
+   - `"Exists"`
+   - `"Gt"`
+   - `"In"`
+   - `"Lt"`
+   - `"NotIn"` 
+-->
+- **operator** (字符串), 必选项
```
Comment (howieyuen) [inline content/zh/docs/reference/kubernetes-api/common-definitions/node-selector-requirement.md:75]:
```suggestion
- **operator** (string)，必选
```

File: content/zh/docs/reference/kubernetes-api/common-definitions/node-selector-requirement.md [hunk 11]
```diff
@@ -0,0 +1,96 @@
...
+
+  operator是表示键与一组值的关系的运算符。有效的运算符包括：In、NotIn、Exists、DoesNotExist、Gt和Lt。
```
Comment (howieyuen) [inline content/zh/docs/reference/kubernetes-api/common-definitions/node-selector-requirement.md:77]:
```suggestion
  表示键与一组值的关系的运算符。有效的运算符包括：In、NotIn、Exists、DoesNotExist、Gt 和 Lt。
```

File: content/zh/docs/reference/kubernetes-api/common-definitions/node-selector-requirement.md [hunk 12]
```diff
@@ -0,0 +1,96 @@
...
+
+  可用的取值:
```
Comment (howieyuen) [inline content/zh/docs/reference/kubernetes-api/common-definitions/node-selector-requirement.md:79]:
```suggestion
  可选值：
```

File: content/zh/docs/reference/kubernetes-api/common-definitions/node-selector-requirement.md [hunk 13]
```diff
@@ -0,0 +1,96 @@
...
+   - `"In"`
+   - `"Lt"`
+   - `"NotIn"`
+
+<!--
+- **values** ([]string)
+
+  An array of string values. If the operator is In or NotIn, the values array must be non-empty. If the operator is Exists or DoesNotExist, the values array must be empty. If the operator is Gt or Lt, the values array must have a single element, which will be interpreted as an integer. This array is replaced during a strategic merge patch.
+-->
+- **values** （字符串数组）
```
Comment (howieyuen) [inline content/zh/docs/reference/kubernetes-api/common-definitions/node-selector-requirement.md:92]:
```suggestion
- **values** （[]string）
```

File: content/zh/docs/reference/kubernetes-api/common-definitions/node-selector-requirement.md [hunk 14]
```diff
@@ -0,0 +1,96 @@
...
+
+   values是一个字符串数组。如果运算符为In或NotIn，则该数组必须为非空。如果运算符为Exists或DoesNotExist，则该数组必须为空。如果运算符为Gt或Lt，则数组必须有一个元素，该元素将被看作整数。该数组会在合入全局补丁时被替换。
```
Comment (howieyuen) [inline content/zh/docs/reference/kubernetes-api/common-definitions/node-selector-requirement.md:94]: proper line break, which makes CR easier and cleaner
```suggestion
  字符串数组。 如果运算符为 In 或 NotIn，则数组必须为非空。
  如果运算符是 Exists 或 DoesNotExist，则数组必须为空。
  如果运算符是 Gt 或 Lt，则数组必须有一个元素，该元素将被译为整数。
  该数组在合并计划补丁是时被替换。
```

**Review by howieyuen — CHANGES_REQUESTED** [review]

File: content/zh/docs/reference/kubernetes-api/common-definitions/node-selector-requirement.md [hunk 15]
```diff
@@ -0,0 +1,99 @@
...
+The file is auto-generated from the Go source code of the component using a generic
+[generator](https://github.com/kubernetes-sigs/reference-docs/). To learn how
+to generate the reference documentation, please read
+[Contributing to the reference documentation](/docs/contribute/generate-ref-docs/).
+To update the reference content, please follow the 
+[Contributing upstream](/docs/contribute/generate-ref-docs/contribute-upstream/)
+guide. You can file document formatting bugs against the
+[reference-docs](https://github.com/kubernetes-sigs/reference-docs/) project.
+-->
+<!--
```
Comment (howieyuen) [inline content/zh/docs/reference/kubernetes-api/common-definitions/node-selector-requirement.md:36]: 译文被注释了，删除36行和41行的 html 注释标签

**Review by howieyuen — CHANGES_REQUESTED** [review]

File: content/zh/docs/reference/kubernetes-api/common-definitions/node-selector-requirement.md (hunk 15 above)
Comment (shixiuguo) [reply content/zh/docs/reference/kubernetes-api/common-definitions/node-selector-requirement.md:36]: 这一段上面的原文就是注释的

**Comment by shixiuguo** [comment]
您好，这一段在英文文档里，原本就是注释的













在 2022-03-07 15:42:35，"Hao Yuan" ***@***.***> 写道：

@howieyuen requested changes on this pull request.

In content/zh/docs/reference/kubernetes-api/common-definitions/node-selector-requirement.md:

> +weight: 5

+auto_generated: true

+---

+-->

+

+

+<!--



译文被注释了，删除36行和41行的 html 注释标签

—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by howieyuen** [comment]
> 您好，这一段在英文文档里，原本就是注释的 在 2022-03-07 15:42:35，"Hao Yuan" ***@***.***> 写道： @howieyuen requested changes on this pull request. In content/zh/docs/reference/kubernetes-api/common-definitions/node-selector-requirement.md:
> +weight: 5
> +auto_generated: true +--- +--> + + +<!-- 译文被注释了，删除36行和41行的 html 注释标签 — Reply to this email directly, view it on GitHub, or unsubscribe. Triage notifications on the go with GitHub Mobile for iOS or Android. You are receiving this because you authored the thread.Message ID: ***@***.***>

抱歉，我看到译文，想当然地认为了。那就把这段原文和译文都删掉，顺便把这些个commit合并，谢谢

**Pull request merged**